packages/opentelemetry-exporter-zipkin/package.json

```diff
@@ -86,7 +86,7 @@
     "webpack-merge": "5.8.0"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.0.3"
+    "@opentelemetry/api": "^1.1.0"
```

Comment:
do we really require 1.1.0 here? Or do we just update to stay on latest?
other packages like e.g. b3 propagator allow `>=1.0.0 <1.2.0` but this extended range vanishes if all exporters require 1.1.0.

Reply:
I'll update it to use the range

Reply:
Since the exporters don't implement any interfaces from the API, should we keep `^1.0.0`?

Reply:
d5b2091acb690b8e0af340669d59ab86debccbbf

Reply:
exporter-trace-otlp-grpc is still on 1.1.0.
Yes, I think `^1.0.0` would be slightly better but I don't think it's needed in this PR.

Reply:
>exporter-trace-otlp-grpc is still on 1.1.0.

oops i didn't save the open file 😆